tests/spec/SpecHelper.js

```diff
@@ -57,6 +57,60 @@ export function makeRSDoc(opts, src, style = "") {
   });
 }
 
+/**
+ * @param {string[]} modules
+ * @return {Promise<Document>}
+ */
+export function makeUnitTestDocument(
+  modules,
+  config = {},
+  { head = `<meta charset="UTF-8" />`, body = "" } = {}
+) {
+  const html = `
+    <!DOCTYPE html>
+    <html lang="en">
+      <head>
+        ${head}
+        <script>
+          var respecConfig = ${JSON.stringify(config || {}, null, 2)};
+        </script>
+        <script type="module">
+          import * as baseRunner from "/base/src/core/base-runner.js";
+          const modules = ${JSON.stringify(modules)};
+          const importedModules = await Promise.all(
+            modules.map(m => import("/base" + m))
+          );
+          await baseRunner.runAll(importedModules);
+        </script>
```

Comment:
Experimenting with "unit testing" modules as "units"...

Reply:
```diff
START:
  Core — data-abbr
    ✔ processes definition abbreviations
    ✔ correctly parses and abbreviates different Abbreviation syntax
    ✔ warns when used with unsupported elements
    ✔ allows different abbreviation combinations to link to dfn
    ✔ normalizes title of added abbr for unwanted spaces

- Finished in 1.335 secs / 1.315 secs
+ Finished in 0.307 secs / 0.294 secs
```

Reply:
Sure, but speed is not a big concern on the testing side, right? `fit(` and `fdescribe(` solve mostly for it during development. But making sure that we catch one plugin breaking/regressing another is more critical, which is usually what happens in practice (when I touch the code 😅).